site/_posts/2021-10-26-how-to-set-up-single-sign-on-between-fusionauth-wordpress.md

```diff
@@ -0,0 +1,199 @@
+
+{% include _image.liquid src="/assets/img/blogs/wordpress-sso-fusionauth/wordpress-initial-screen.png" alt="Choosing a language for WordPress installation." class="img-fluid box-shadow" figure=false %}
+
+Choose your preferred language and click "Continue.”
+
+On the next page, enter the site title and create an admin account by providing a username, email, and password.
+
+{% include _image.liquid src="/assets/img/blogs/wordpress-sso-fusionauth/wordpress-admin-setup-screen.png" alt="Creating a WordPress administration account." class="img-fluid" figure=false %}
+
+Press "Install WordPress" to finish the installation. Log in with your new username and password when it's finished.
+
+### Configuring FusionAuth
+
+Now let's set up FusionAuth. Navigate to `localhost:9011`. You will see the FusionAuth setup screen.
+
+{% include _image.liquid src="/assets/img/blogs/wordpress-sso-fusionauth/setup-wizard.png" alt="Setting up FusionAuth." class="img-fluid" figure=false %}
+
+Enter your first name, last name, email, and password. Once done, click on "Submit" to finish the installation.
+
+## Adding FusionAuth SSO to WordPress
+
+Once FusionAuth and WordPress have been installed, you can proceed with adding FusionAuth SSO to your WordPress instance. In this section, you will install an OAuth Single Sign-On plugin and configure it to communicate with your local FusionAuth instance.
+
+### Installing the plugin
+
+For this tutorial, you will use the [OpenID Connect Generic plugin](https://github.com/oidc-wp/openid-connect-generic).
+
+Go to your WordPress Admin Dashboard at `localhost:8030/wp-admin` and click on the "Plugins" page from the sidebar. Click on the "Add New" button, and on the next page, search for "OpenID Connect Generic".
+
+The plugin should be the first one in the search results. You want the one by `daggerhart`. Install and activate the plugin.
+
+{% include _image.liquid src="/assets/img/blogs/wordpress-sso-fusionauth/select-plugin.png" alt="Selecting the WordPress OIDC plugin." class="img-fluid" figure=false %}
+
+Once the plugin is activated, choose "Settings" and then "OpenID Connect Client" from the left menu. This is where you will be able to configure the plugin to work with FusionAuth.
+
+{% include _image.liquid src="/assets/img/blogs/wordpress-sso-fusionauth/configure-oidc-generic-client-settings-start.png" alt="Configuring the WordPress OIDC plugin." class="img-fluid" figure=false %}
+
+But first, let's set up FusionAuth. After you do that, you'll return to finish configuring this WordPress plugin.
+
+### Configuring FusionAuth
+
+To get started with SSO, you need to create an application in FusionAuth. An application represents the resource where users will log in to, in your case, the WordPress site. Each website, mobile app, or any other application should be created as an application in FusionAuth. Users can use the same username and password to log in to all such applications managed by FusionAuth.
```

Comment:
I'd go into a bit more here of why users are logging into my word press site and not my original site - my first question was "wait, what happened to my photo-sharing app, and why isn't that the one being used for SSO?" May be good to demonstrate why WP is the SSO site and not the photo sharing app

Reply:
I tried to explain that a bit more.